Sources/DBUS/DBusUnixFDClient.swift

```diff
@@ -0,0 +1,632 @@
+import Dispatch
+import Logging
+import NIOCore
+
+#if canImport(FoundationEssentials)
+  import FoundationEssentials
+#elseif canImport(Foundation)
+  import Foundation
+#endif
+
+#if canImport(Glibc)
```

Comment:
MUSL too

Reply:
We can just `#if os(Linux)` for the DbusUnixFDClient here and then import Musl or Glibc - whichever is available